java-compiler-testing/src/test/java/io/github/ascopes/jct/tests/unit/filemanagers/impl/JctFileManagerImplTest.java

```diff
@@ -76,4 +78,18 @@ void testAddPathForOutputLocation() {
     assertThat(jctFileManager.hasLocation(outputLocation)).isTrue();
   }
 
+  @Test
+  @DisplayName("Tests addPath method for adding module location")
+  void testAddPathForModuleLocation() {
+    var moduleLocation = StandardLocation.locationFor(StandardLocation.MODULE_PATH.name());
+    var pathRoot = mock(PathRoot.class);
+    var path = Paths.get(".").normalize().toAbsolutePath();
+
+    // we mock path because it is needed by AbstractPackageContainerGroup
+    given(pathRoot.getPath()).willReturn(path);
+
+    JctFileManagerImpl jctFileManager = JctFileManagerImpl.forRelease("test");
+    jctFileManager.addPath(moduleLocation, pathRoot);
+    assertThat(jctFileManager.hasLocation(moduleLocation)).isEqualTo(true);
```

Comment:
should be able to just say .isTrue rather than .isEqualTo(true) here to keep it a little more concise

Reply:
Will do